### ISSUE TITLE
AC not completed after more than 3000 attempt

### ISSUE THREAD

**Issue by Hydrog3n** [brave/brave-browser#11794]
## Description 

It's happened last month and this month again my uphold wallet is withdraw but the process still try and try in brave://rewards-internals

## Steps to Reproduce

No idea

## Actual result:

![image](https://user-images.githubusercontent.com/1223893/93815488-5748d780-fc56-11ea-97a9-f1513a26bb1b.png)

I have 14 contributions the two last have 0 has "contribution amount" 

![image](https://user-images.githubusercontent.com/1223893/93815563-76e00000-fc56-11ea-8c82-bce33f4fe62b.png)

## Expected result:
Working 

## Reproduces how often: 

Since 2 AC (2 months)

## Brave version (brave://version info)

Brave | 1.14.81 Chromium: 85.0.4183.102 (Build officiel) (64 bits)
-- | --
Révision | ffe848af6a5df4fa127e2929331116b7f9f1cb30-refs/branch-heads/4183@{#1770}
Système d'exploitation | Linux

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? I don't know
- Does the issue resolve itself when disabling Brave Rewards? No

**Comment by Miyayes** [comment]
Has @LaurenWags or other testers seen this as of late?

**Comment by LaurenWags** [comment]
Thanks for reporting back @Hydrog3n! I do not think any of the older ones will complete their contribution, but am not 100% certain.

@Miyayes I haven't seen any of these types of issues in a long while. We fixed a statistical voting issue (#15071) that I think might have helped. Since the OP has said this is now working for them, maybe we could close this issue and open a new one if we encounter this again.

**Comment by Hydrog3n** [comment]
> Thanks for reporting back @Hydrog3n! I do not think any of the older ones will complete their contribution, but am not 100% certain.

Yes but it's more it continues to try to use process and compute maybe. If you have a solution to stop try to be processed can be nice. Like after x times try they failed.
